<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CanOpenUI</class>
 <widget class="QMainWindow" name="CanOpenUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>712</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="ModBusView">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>341</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;协议接收&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>20</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;发送指令&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>270</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>CanOpen  IP:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>270</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Port:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>70</y>
      <width>341</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="CanIP">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>270</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>127.0.0.1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="CanPort">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>270</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>502</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CanConn">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>300</y>
      <width>281</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>连接</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>270</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Modbus Master Port:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ModPort">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>270</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>502</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ModConn">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>300</y>
      <width>201</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>开启协议接收</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendCan">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>330</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QWidget" name="GLSlot" native="true">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>370</y>
      <width>690</width>
      <height>270</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
